src/components/Identity.vue

```diff
@@ -1,6 +1,12 @@
 <template>
```

Comment:
I went through this pull request by commit. I feel like this commit marks a spot in this branch that deserves its own pull request - to such a degree that I might be tempted to carve them off of this branch and resubmit them as a new pull request without the changes to the database env vars, icons, &c included. Thoughts?

Reply:
This PR is indeed two-in-one; I foolishly buried a commit I needed to run the dev environment on Windows (e5807ae) in the middle of my previous PR, #126, so all the commits from that are here as well, because I didn't know how to just keep that particular one; I would appreciate any feedback on what a better approach would have been, once I'd realized my mistake; aside from, obviously, further perusing the docs for Git. Feel free to eliminate the commits that should just be in #126 from this PR; if worst comes to worst, I can just copy and paste the contents of e5807ae into this branch and commit that change a second time.

Reply:
I can definitely help with that. I'm on a call for another hour or so, but I'm happy to write up some ideas on how to separate off these two commits.

Reply:
I did my best to do it with just `git rebase -i HEAD~11` and a force push; I also wouldn't mind hearing about other ways to accomplish this task, and I also wouldn't mind if you checked to make sure I haven't reduced any obscure yet important property of this Git branch to smithereens

Reply:
I had planned on writing up something longer whose punch line was basically `git rebase -i HEAD~some numbers` anyways, so you're clearly doing great already. Looking at the results, I think you got it pretty much perfect: all the changes prior to the one that introduces changes in `Identity.vue` are still present on `save-button`, but are no longer present on this branch. That makes it much, much easier to review this, and in fact obviates some of my comments entirely, without any obvious lost work.

Given how notorious Git is for causing people to lose work the first few times they try to use rebase, I'm very impressed. What's your mental model of what Git did with this?

Reply:
Double edit: was leaving the "quick database login shim for windows" commit in this branch a deliberate choice? It's now duplicated with an identical commit in your `save-button` branch.

Reply:
Well, I've developed a decent mental model of how commits work; like, they apply a diff, which basically means replacing specific lines with specific other lines, or adding them, or deleting them; and I have a decent mental model of a PR as a chain of commits, each based on the version of the codebase created by the last; so when I googled and found `git rebase -i` I was pretty sure I understood what removing commits would do; alter the chain of changes. I regret to inform you that those changes should not be on `save-button` either though, just my `master` branch, which forms the basis for #126, whoops

Reply:
Pretty much spot on. Git's snapshot-based rather than diff-based under the hood, but the two models are equivalent and your analysis is exactly right as a result. Nice work.

Reply:
Thank you. I think I've got `save-button` (the basis for #128) under control too, although it now appears to give me undeserved credit for a few old commits:
![image](https://user-images.githubusercontent.com/49729978/74897971-8f0d3b00-5366-11ea-9689-c4063781be7a.png)
So hopefully that'll teach me to actually figure out what value to use for `X` in `git rebase i- HEAD~X` instead of just guessing, next time...

Reply:
>Double edit: was leaving the "quick database login shim for windows" commit in this branch a deliberate choice? It's now duplicated with an identical commit in your save-button branch.

Yes, I need that commit for the database connection to work at all on my machine, as I wrote when I initially dropped it in [here](https://github.com/wlonk/wheretofind.me/pull/126#discussion_r379660098)

Reply:
That's probably due to those commits being merged into your branch somewhere along the line. I have a previous version of the `save-button` branch locally from looking at this PR; I can see that 37920f1 (Bump django from 2.2.8 to 2.2.10) is in its history in complicated ways.

One way to avoid this is to let `git rebase` work out where to rebase from, rather than counting up the numbers yourself. `git rebase -i origin/master` will open a rebase editor with every commit on your branch going all the way back to wherever it diverges from `origin/master` (formally: every commit reachable from HEAD which is not also reachable from origin/master).

Since you're already in the situation of having accidentally duplicated some commits, one way to fix this might be to run `git rebase -i origin/master` and to edit the duplicated commits out of the branch. _DO THIS ON A SCRATCH BRANCH FIRST_ to look at the results before doing it "for real" - `git checkout -b rebase-experiment-1` first, or something, and `git branch -D rebase-experiment-1` when you're done trying it out.

Reply:
>_DO THIS ON A SCRATCH BRANCH FIRST_

This was good advice, because I just tried this and it did some fairly complicated things (it didn't show the duplicated commits at all in the commits list text file window, and then it automatically re-integrated all the commits I just removed from `save-button` back in when I therefore just closed it). Maybe I'll just keep taking credit for those random commits for a while and hope that poor @wlonk can somehow figure out how to merge those two branches if he decides to use the PR.